spec/velveteen/worker_spec.rb

```diff
@@ -64,6 +64,61 @@
       _, properties, _ = queue.pop
       expect(properties[:headers]).to eq("foo" => "bar")
     end
+
+    describe "payload validation" do
+      it "raises an error when payload does not match schema" do
+        message = Velveteen::Message.new(
+          data: {"foo" => "bar"},
+          headers: {}
+        )
+        invalid_message = Velveteen::Message.new(
+          data: {"coffee" => "espresso"},
+          headers: {}
+        )
+        publish_routing_key = "worker.test.tea.ordered"
```

Comment:
It looks like both specs use the same routing key. What do you think about the test worker hard coding that key and removing `publish_routing_key` as an argument?

Reply:
into it